apps/minifront/src/components/v2/dashboard-layout/transactions-card-title.tsx

```diff
@@ -3,22 +3,26 @@ import { CharacterTransition } from '@repo/ui/CharacterTransition';
 import { Dialog } from '@repo/ui/Dialog';
 import { Text } from '@repo/ui/Text';
 import { Info } from 'lucide-react';
+import { useId } from 'react';
 
-export const TransactionsCardTitle = () => (
-  <div className='flex items-center gap-2'>
-    <CharacterTransition>Transactions List</CharacterTransition>
-    <Dialog>
-      <Dialog.Trigger asChild>
-        <Button icon={Info} iconOnly='adornment'>
-          Info
-        </Button>
-      </Dialog.Trigger>
-      <Dialog.Content title='Transactions List'>
-        <Text>
-          Your wallet scans shielded chain data locally and indexes all relevant transactions it
-          detects, both incoming and outgoing.
-        </Text>
-      </Dialog.Content>
-    </Dialog>
-  </div>
-);
+export const TransactionsCardTitle = () => {
```

Comment:
You may want to review this file with whitespace changes hidden. I just added a `useId()` call, and then passed it to the dialog trigger and content to make the dialog "grow out of" the info icon.